assets/Langchain/environments/general-langchain-app-deployment/environment.yaml

```diff
@@ -0,0 +1,12 @@
+image:
+  name: azureml/curated/general-langchain-app-deployment
```

Comment:
you need to onboard image with MCR and vulnerability scans before this can go in

Reply:
new two PRs are out.

Reply:
The two PRs are merged.